test/jdk/java/io/Bits/ReadWriteValues.java

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @summary Verify that reads and writes of primitives are correct
+ * @compile/module=java.base java/io/BitsProxy.java
+ * @run junit/othervm --add-opens java.base/java.io=ALL-UNNAMED ReadWriteValues
+ */
```

Comment:
I see you've figured out to test with a helper classes in java.io, good! 

Did you mean to add a test tag to BitsProxy? That will cause jtreg to treat it as a test in its own right and it will fail.

For ReadWriteValues, the --add-opens isn't needed as the test no longer does any deep reflection on APIs in that package. You can add a bug tag too, which should give you:

```
/*
 * @test
 * @bug 8299576
 * @summary Verify that reads and writes of primitives are correct
 * @compile/module=java.base java/io/BitsProxy.java
 * @run junit ReadWriteValues
 */
```